htdocs/app/Migrations/Version20171008183000.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Class Version20171008183000
+ *
+ * @package Application\Migrations
+ */
+class Version20171008183000 extends AbstractMigration
+{
+    /**
+     * Sets the engine of the table user to INNODB.
+     *
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE user ENGINE = INNODB;');
```

Comment:
no innodb at the moment. Have a look into our board. https://forum.opencaching.de/index.php?topic=4762.msg61245#msg61245

Reply:
Innodb is active since a couple of months without any issue :-)